### DISCUSSION TITLE
Auto acknowledgement On/Off Option

### DISCUSSION THREAD

**Discussion by vipin9387** [rabbitmq/rabbitmq-server#9858] [Other] [answered]
### Is your feature request related to a problem? Please describe.

I am using KEDA (Kubernetes Event Driven Autoscaling) to deploy my Go binary that consumes payloads from a RabbitMQ queue. My concern is that when I insert 300 messages in the queue, KEDA will scale the first pod to consume 10 messages. However, all the messages are being inserted into the RabbitMQ queue in an "unacked from ready" state. To understand what's happening, I want to test the effect of turning auto-acknowledgement on or off from the console. However, I have not been able to locate this option in the RabbitMQ console. I am currently using version 3.8.35 of RabbitMQ.

### Describe the solution you'd like

How to enable or disable auto-acknowledgement option

**Comment by michaelklishin** [comment, marked as answer]
Acknowledgement mode and the prefetch setting are [application-controlled](https://rabbitmq.com/confirms.html). There are no RabbitMQ configuration settings or policies that control them.

Tutorial 2 [provides an introduction](https://rabbitmq.com/getstarted.html) to the concepts and features code examples for many popular RabbitMQ clients in different programming languages.

FYI, RabbitMQ 3.8 has [reached end of life over one year ago](https://rabbitmq.com/versions.html).

**Comment by michaelklishin** [comment]
Please use GitHub discussions for questions in the future. They are no longer a brand new, mostly unknown GitHub feature.
